domain/schema/controller/triggers/cloud-triggers.gen.go

```diff
@@ -8,7 +8,6 @@ import (
 	"github.com/juju/juju/core/database/schema"
 )
 
-
```

Comment:
Yea, I had been using the experimental free version because my subscription for Goland ran out and I was waiting for a new one. I was also having to run `go fmt` manually every time. Should be fixed now.

I've redone the merge dropping these changes.